README.md

```diff
@@ -172,6 +172,13 @@ libraryDependencies ++= Seq(
 
 where `<version>` is the latest on [maven central](http://search.maven.org/#search|ga|1|g:org.scalaz%20a:scalaz-deriving_2.12).
 
+If you are supplying a `deriving.conf` file, make sure the following is added to your project settings, so that the `deriving.conf` is present
+on the compilation classpath:
+
```

Comment:
I am having a hard time deciding whether this should be a separate paragraph in the doc or be added in the previous code listing. It's technically unneeded for anyone not using deriving.conf. Then again if you change your project from not using to using, you can get nasty surprises courtesy of sbt - specifically due to the nondeterminism. The maintainers will know best.